src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -146,6 +151,15 @@ The .NET Foundation licenses this file to you under the MIT license.
       <ObjCopyNameAlternative Condition="$(CppCompilerAndLinker.Contains('clang'))">objcopy</ObjCopyNameAlternative>
     </PropertyGroup>
 
+<!-- objwriter and jittools are built under LLVM12; FreeBSD's ports of llvm## name clang as clang## with objcopy as llvm-objcopy##
+     the system "clang" and "objcopy" are likely not the same version as the ports version but are left available as a fallback to smoke tests which seems to default to "clang" -->
+    <PropertyGroup Condition="'$(ObjCopyName)' == '' and '$(TargetOS)' == 'FreeBSD'">
```

Comment:
We are avoiding hard-coded versions in this (shipping) targets. If you have confirmed that smoke tests on main branch fail without it, you can override with
`<ObjCopyName>llvm-obcopy12</ObjCopyName>` after: https://github.com/dotnet/runtime/blob/5fdc1882a63dd1e34ad852f9bfc1964c511e5bfb/eng/testing/tests.singlefile.targets#L27

Reply:
>  If you have confirmed that smoke tests on main branch fail without it

IIRC, smoke tests don't need `objcopy`, so you can just delete it. `objcopy` is used for symbol stripping:
`dotnet publish -p:PublishAot=true -p:StripSymbols=true`. So if debug symbols are really broken for those users without explicitly specifying llvm-objcopy-12, they can pass `-p:ObjCopyName=llvm-objcopy-##`

Reply:
>  If you have confirmed that smoke tests on main branch fail without it

IIRC, smoke tests don't use `objcopy`, so you can just delete it. `objcopy` is used for symbol stripping:
`dotnet publish -p:PublishAot=true -p:StripSymbols=true`. So if debug symbols are really broken for those users without explicitly specifying llvm-objcopy-12, they can pass `-p:ObjCopyName=llvm-objcopy-##`